# This message is used to assign values to all the servos at once

float32 servo5
float32 servo6
float32 servo7
float32 servo8
float32 servo9
float32 servo10
float32 servo12
float32 servo13
float32 servo20
float32 servo21
float32 servo22
float32 servo27
